docs/reference/modules/transport.asciidoc

```diff
@@ -132,6 +132,8 @@ transport.profiles.dmz.bind_host: 172.16.1.2
 The `default` profile is special. It is used as a fallback for any other
 profiles, if those do not have a specific configuration setting set, and is how
 this node connects to other nodes in the cluster.
+The other profiles can have any name, and can be used to set up specific 
+endpoints for incoming client connections.
```

Comment:
Minor grammatical nit
```suggestion
Other profiles can have any name and can be used to set up specific endpoints
for incoming client connections.
```

Reply:
I'd rather drop the word `client` too, the only client using the transport port is the (deprecated) transport client. These days the transport profiles are mostly useful for exposing different endpoints for remote cluster connections. Either `incoming transport connections` or just `incoming connections` would be better IMO.